rwmutex.go

```diff
@@ -113,8 +124,20 @@ func (m *RWMutex) TryRLock() bool {
 // It is a run-time error if rw is not locked for reading
 // on entry to RUnlock.
 func (m *RWMutex) RUnlock() {
-	m.wg.Done()
-	m.mutex.RUnlock()
+	defer m.mutex.RUnlock()
+
+	if DebugIsOn {
+		m.wg.Done()
+		defer func() {
+			err := recover()
+			if err != nil {
+				// when m.wg goes bellow zero,
+				// we need to recover from panic for the unit tests,
+				// re-init the wg
```

Comment:
```suggestion
				// when the WaitGroup goes below zero,
				// we need to recover from panic,
				// and re-initialize it
```
suggestion for a tiny phrasing fix, not sure where the 80 characters limit is 😄

Reply:
sorry for my French